src/Microsoft.DotNet.Darc/DarcLib/ILocalGitRepo.cs

```diff
@@ -124,4 +124,16 @@ public interface ILocalGitRepo : IGitRepo
         string repoPath,
         string remoteUri,
         CancellationToken cancellationToken = default);
+
+    /// <summary>
+    /// Retrieves SHA of a commit that last changed a given line in a given file.
```

Comment:
```suggestion
    /// Retrieves SHA of the commit that last changed the given line in the given file.
```
To match the line below.